ee/clickhouse/queries/trends/breakdown.py

```diff
@@ -95,7 +97,7 @@ def get_query(self) -> Tuple[str, Dict, Callable]:
             "parsed_date_to": parsed_date_to,
             "actions_query": "AND {}".format(action_query) if action_query else "",
             "event_filter": "AND event = %(event)s" if not action_query else "",
-            "filters": prop_filters if props_to_filter else "",
+            "filters": prop_filters if props_to_filter.groups else "",
```

Comment:
Hmm. Took me a second to grasp what this value `.groups` is. Perhaps we shouldn't introduce that verb at all here (since it causes confusion with e.g. group analytics) and instead name it `values` or `property_groups`?

Reply:
agreed!

Reply:
Yeahh, definitely.

Reply:
(will do in a follow up, makes other things hard to parse in here)